### PR TITLE
First whack at reg support.

### PR DESCRIPTION
**Pull request by jaymzh**
This isn't tested at all yet.

### CODE REVIEW

**Comment by leizleiz** [comment]
Maybe backup the hand written configs that will be affected by this, then commit this, let chef do its thing, and sanity check the result?

File: cookbooks/scale_reg/recipes/default.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  },
+}
+
+node.default['fb_apache']['sites']['*:80'] = vhost_config
+{
+  'ErrorLog' => '/var/log/httpd/error.log',
+  'CustomLog' => '/var/log/httpd/access.log combined',
+}.each do |key, val|
+  node.default['fb_apache']['sites']['*:80'][key] = val
+end
```
Comment (irabinovitch) [inline cookbooks/scale_reg/recipes/default.rb]: can we add an 80->443 redirect ?

**Comment by irabinovitch** [comment]
@jaymzh @leizleiz anything else pending here?

File: cookbooks/scale_reg/recipes/default.rb [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+#
+# All rights reserved - Do Not Redistribute
+#
+
+package 'python-mysql' do
+  action :upgrade
+end
+
+directory '/var/www' do
+  owner 'root'
```
Comment (jaymzh) [inline cookbooks/scale_reg/recipes/default.rb]: I don't believe that's true on our CentOS images - it hasn't been an issue on www.

File: cookbooks/scale_reg/recipes/default.rb (hunk 1 above)
Comment (jaymzh) [inline cookbooks/scale_reg/recipes/default.rb]: sure

**Comment by jaymzh** [comment]
I need to test this and then actually backup existing stuff and roll it out... I hope to do that this weekend, sorry with chef summit and halloween and houseguests it's been crazy.

**Pull request merged**